tests/e2e/allowlists/allowlists_service_invocation_test.go

```diff
@@ -88,25 +89,26 @@ var allowlistsServiceinvocationTests = []struct {
 }
 
 var moreAllowlistsServiceinvocationTests = []struct {
```

Comment:
Can you rename `moreAllowlistsServiceinvocationTests` to be more specific ?

I would refactor `allowlistsServiceinvocationTests` and combine this to `allowlistsServiceinvocationTests`. 

I can see the duplicated test code too. please remove duplicated codes and combine it to single one.

Reply:
fixed